client/post-editor/post-editor.jsx

```diff
@@ -207,7 +211,7 @@ const PostEditor = React.createClass( {
 		PostEditStore.on( 'change', this.onEditedPostChange );
 		this.debouncedSaveRawContent = debounce( this.saveRawContent, 200 );
 		this.debouncedAutosave = debounce( throttle( this.autosave, 20000 ), 3000 );
-		this.recordedDefaultEditorMode = false;
+		this.recordedDefaultEditorModeFromPreferences = false;
```

Comment:
Why the name change here? Do we even use this? I could only find a single reference in the code.